### ISSUE TITLE
Add sparse vectors to migration tests

### ISSUE THREAD

**Issue by joein** [qdrant/qdrant-client#446]
https://github.com/qdrant/qdrant-client/blob/efb876fe3915dc5e2855f60a5617e940c84591e5/tests/test_migrate.py#L67

**Comment by shivas1516** [comment]
I've reviewed the code and believe I can enhance the migration test by adding sparse vectors and already started working on it. If you have any thoughts or suggestions lemme know

**Comment by shivas1516** [comment]
I encountered following issue while adding sparse_vector to the migration tests.

I made some adjustments to the migration tests to included sparse vectors in the points in the below image. 
Changes ai did:
![Screenshot 2](https://github.com/qdrant/qdrant-client/assets/141582345/b6b0f1c2- [ ] -a806-46ad-9985-ef63bc2b2324.png)

After running the tests locally, I encountered 3 failures
Failures of the test:
![Screenshot 1](https://github.com/qdrant/qdrant-client/assets/141582345/81b22bb7-6634-469e-89e4-85b6ead1b38f.png)

I suspect there might be an issue related to the sparse vectors or project setup.

Suggestions or guidance to resolve this issue are most welcome
